README.md

```diff
@@ -73,6 +73,16 @@ jobs:
           fail: true
 ```
 
+You may want to add additional arguments to the above.  In particular if you're testing a site directly from the file-system (as in the above), you'll likely want to set the argument `--base .` to ensure that **all links** in the files are tested.  You don't need to do this if you're testing a hosted site.
+
+```
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.8.0
+        with:
+          fail: true
+          args: "--base . --verbose --no-progress './**/*.md' './**/*.html' './**/*.rst'"
```

Comment:
```suggestion
          args: --base . --verbose --no-progress './**/*.md' './**/*.html' './**/*.rst'
```

Although it's not required, removing double quotes (since this is a string) makes it simpler.